src/api/controllers/articleController.js

```diff
@@ -56,6 +57,67 @@ export default class ArticleController {
     }
   }
 
+  /**
+   * This function gets all the articles that are stored in the article table
+   *
+   * @static
+   * @param {*} req the request
```

Comment:
@truestbyheart this error is on all of your PRs.
The req parameter is an object